File: app/src/main/java/com/example/kotlin/mvc/java/base/view/IBaseTargetView.java
package com.example.kotlin.mvc.java.base.view;

import com.example.kotlin.mvc.java.base.module.BaseModule;

/**
 * 使用对象：Fragment Activity
 */
public interface IBaseTargetView {

    void addModule(BaseModule module);
    void removeModule(BaseModule module);

}

File: app/src/main/java/com/example/kotlin/mvc/kotlin/base/view/IBaseTargetView.kt
package com.example.kotlin.mvc.kotlin.base.view

import com.example.kotlin.mvc.kotlin.base.module.BaseModule

/**
 * 使用对象：Fragment Activity
 */
interface IBaseTargetView {

    fun addModule(module: BaseModule<*, *>?)
    fun removeModule(module: BaseModule<*, *>?)

}